bundle/android/strings.xml

```diff
@@ -309,9 +313,6 @@ translators: %s: Select control button label e.g. "Button width" -->
     <string name="gutenberg_native_replace_video" tools:ignore="UnusedResources">Replace video</string>
     <string name="gutenberg_native_retry" tools:ignore="UnusedResources">Retry</string>
     <string name="gutenberg_native_rich_text_editing" tools:ignore="UnusedResources">Rich text editing</string>
-    <!-- translators: %s: Block name e.g. "Image block"
```

Comment:
This is [still here](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/autocomplete/autocompleter-ui.native.js#L187C1-L188C1) not sure why it was removed 🤔